### PR TITLE
Makes F-list's image host provide headshots

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request

Currently, only two websites are accepted by the headshot regex. This fixes a problem that is occurring with Discord right now, where images have started being deleted after a certain amount of time.

## Why It's Good For The Game

Gyazo sucks as an image host, so while it does technically serve its purpose, we want to expand our pool of potential options. F-list properly verifies that what it receives is an image (furries winning yet again) and so it should fit our criteria for new headshots.

## Changelog

:cl: ReturnToZender (code)
add: F-list headshots are now accepted
/:cl:

### CODE REVIEW

**Review by Majkl-J — APPROVED** [review]
Gyattzo.

**Comment by YakumoChen** [comment]
Lensdump is also supported

File: modular_skyrat/master_files/code/modules/client/preferences/headshot.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	maximum_value_length = MAX_MESSAGE_LEN
 	/// Assoc list of ckeys and their link, used to cut down on chat spam
 	var/list/stored_link = list()
-	var/static/link_regex = regex("i.gyazo.com|media.discordapp.net|cdn.discordapp.com|a.l3n.co|b.l3n.co|c.l3n.co") //gyazo, discord, lensdump
+	var/static/link_regex = regex("i.gyazo.com|media.discordapp.net|cdn.discordapp.com|a.l3n.co|b.l3n.co|c.l3n.co|static.f-list.net/images/") //BUBBER EDIT: adds F-list's image host to the valid list
```
Comment (YakumoChen) [inline modular_skyrat/master_files/code/modules/client/preferences/headshot.dm:10]: You should still include the other hosts for clarity

Comment (ReturnToZender) [reply modular_skyrat/master_files/code/modules/client/preferences/headshot.dm]: Updated the comment, should be better

**Pull request merged**